Sources/FluentUI_iOS/Components/Notification/FluentNotification.swift

```diff
@@ -101,6 +105,7 @@ public struct FluentNotification: View, TokenizedControlView {
                 actionButtonTitle: String? = nil,
                 actionButtonAction: (() -> Void)? = nil,
                 showDefaultDismissActionButton: Bool? = nil,
+                defaultDismissButtonAction: (() -> Void)? = nil,
```

Comment:
Do we need to think about ay of the Objective-C/UIKit API for this, or are we good with adding it for SwiftUI only for now?

Reply:
Objective C callers should be able to configure this property through the `MSFNotificationState` property exposed on the `MSFNotification`.

Reply:
Resolving this for now so the change can be merged. Happy to continue this conversation and follow up with another change if we want to improve the API in the future.